core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -4541,15 +4541,14 @@ LogicalProject(EXPR$0=[1])
             <![CDATA[
 LogicalProject(EXPR$0=[1])
   LogicalJoin(condition=[=($0, $8)], joinType=[inner])
-    LogicalFilter(condition=[>($0, 7)])
-      LogicalUnion(all=[true])
-        LogicalProject(DEPTNO=[$7])
-          LogicalFilter(condition=[>($7, 7)])
-            LogicalTableScan(table=[[CATALOG, SALES, EMP]])
-        LogicalProject(DEPTNO=[$7])
-          LogicalFilter(condition=[>($7, 10)])
-            LogicalTableScan(table=[[CATALOG, SALES, EMP]])
-    LogicalFilter(condition=[>($7, 7)])
+    LogicalUnion(all=[true])
+      LogicalProject(DEPTNO=[$7])
+        LogicalFilter(condition=[>($7, 7)])
+          LogicalTableScan(table=[[CATALOG, SALES, EMP]])
+      LogicalProject(DEPTNO=[$7])
+        LogicalFilter(condition=[>($7, 10)])
+          LogicalTableScan(table=[[CATALOG, SALES, EMP]])
+    LogicalFilter(condition=[OR(>($7, 7), >($7, 10))])
```

Comment:
Does that mean "simplifying was not applied here"?

Reply:
This testcase now changes back to the pre-CALCITE-2247 state..I forgot to note that.
This is because pullup logic is not using that logic anymore (to avoid perf degradation)